src/webview/webViewEventHandlers.js

```diff
@@ -187,12 +188,21 @@ const handleLongPress = (
   if (!message) {
     return;
   }
-  const { dispatch, showActionSheetWithOptions, backgroundData, narrow } = props;
-  showActionSheet(target === 'header', dispatch, showActionSheetWithOptions, _, {
-    backgroundData,
-    message,
-    narrow,
-  });
+  const { dispatch, showActionSheetWithOptions, backgroundData, narrow, onEditStart } = props;
+  showActionSheet(
+    target === 'header',
+    showActionSheetWithOptions,
+    {
+      dispatch,
+      startEdit: onEditStart,
```

Comment:
Let's pick a single name for this callback :slightly_smiling_face: Makes it easier to trace things through the various layers.

I'm happy leaving out the `on`. I think the name can be improved further by adding the word "message", so it's clear this is about editing a message -- vs. e.g. editing a draft. Perhaps `startEditMessage`, then.

Reply:
Used `startEditMessage` everywhere.
Also changed `onEditComplete` to `completeEditMessage` for consistency.